artist.py

```diff
@@ -0,0 +1,72 @@
+import pygame
+import os
+from common_constants import Vector
+
+# in charge of drawing Game objects
+class Artist:
+    def __init__(self, game):
+        self.game = game
+
+        # pygame setup
+        os.environ['SDL_VIDEO_WINDOW_POS'] = "%d,%d" % (0,0) # locate the window in the upper left
+        pygame.init()
+        pygame.display.set_caption(f'{type(game).__name__} Reduction')
+
+        # compute constants
+
+        # this is the approximate height of the menu bar at the top of iOS
```

Comment:
Why is iOS relevant?

Reply:
on my mac i had to add this buffer because the dimensions of my display returned by pygame did not exactly correlate to the actual usable height of my display because of the menu bar at the top